src/Service/SQS.service.js

```diff
@@ -51,6 +50,42 @@ export default class SQSService extends DependencyAwareClass {
     }
   }
 
+  /**
+   * Returns an SQS client instance
+   */
+  // eslint-disable-next-line class-methods-use-this
+  get sqs() {
+    if (!this.$sqs) {
+      this.$sqs = new AWS.SQS({
+        region: process.env.REGION,
+      });
+    }
+
+    return this.$sqs;
+  }
+
+  /**
+   * Returns a Lambda client instance
+   */
+  // eslint-disable-next-line class-methods-use-this
+  get lambda() {
+    if (!this.$lambda) {
+      const endpoint = process.env.SERVICE_LAMBDA_URL;
+
+      if (!endpoint) {
+        throw new Error('process.env.SERVICE_LAMBDA_URL must be defined.');
+      }
+
+      // move to subprocess
+      this.$lambda = new AWS.Lambda({
+        region: process.env.AWS_REGION,
+        endpoint,
+      });
+    }
+
+    return this.$lambda;
+  }
```

Comment:
Move to a cached getter so:

1. We don't init Lambda on file import
2. We can mock it in tests